actionpack/lib/action_controller/metal/redirecting.rb

```diff
@@ -227,7 +227,7 @@ def _compute_redirect_to_location(request, options) # :nodoc:
     #     url_from("http://example.com/profile")  # => "http://example.com/profile"
     #     url_from("http://evil.com/profile")     # => nil
     #
-    # Subdomains are considered part of the host:
+    # Subdomains are included in the evaluation of the host and are not considered internal.
```

Comment:
The new wording `“Subdomains are included in the evaluation of the host and are not considered internal”` is confusing. It could be read as if Rails treats all subdomains as part of the trusted host, which isn’t correct.

For example, if the configured host is `example.com`, a request to `foo.example.com` is not automatically trusted and will return `nil`, which is part of the host authorization security mechanism.

Given that, I find the existing docs clearer on this point, so I’d suggest leaving them as is and closing this PR.

Reply:
I opened the PR because I found the existing wording confusing. On first read it implied that the subdomain is part of the same host therefore internal and should be trusted. The existing wording does nothing to convey that a subdomain is not part of the same trusted host. 

You don't actually explain how the suggested wording is supposed to be confusing. The existing wording says it's part of the same host which has two meanings. It can mean the server itself which would imply it is internal or as it is intended in this case the specific portion of the URL in question. 

At the end of the day I have very little investment in this and was just hoping to help the next poor soul reading this documentation. If you don't see a problem I'm not going to waste any more energy on this. ✌️